rdr_service/genomic/genomic_job_components.py

```diff
@@ -394,11 +397,26 @@ def _ingest_gc_manifest(self, data, _site):
                     member.genomicWorkflowStateModifiedTime = clock.CLOCK.now()
 
                 self.member_dao.update(member)
+                count += 1
 
                 # Update member for PDR
                 bq_genomic_set_member_update(member.id, project_id=self.controller.bq_project_id)
                 genomic_set_member_update(member.id)
 
+            # Update the record count with the number of ingested records.
+            manifest_file = self.manifest_dao.get(self.file_obj.genomicManifestFileId)
+
+            # TODO: AW1F files won't have a manifest record,
+            #  this will be added in a follow-up PR
+            if manifest_file is not None:
+                # We want to add the record count to the existing record count for that manifest record
+                manifest_file.recordCount += count
+                manifest_file.processingComplete = 1
+                manifest_file.processingCompleteDate = clock.CLOCK.now()
+
+                with self.manifest_dao.session() as s:
+                    s.merge(manifest_file)
+
```

Comment:
Should calls be made to update BQ/Resource to update the ManifestFile there?

Reply:
Yes, good catch. I will add those calls here.